opensrp-core/src/main/java/org/smartregister/repository/EventClientRepository.java

```diff
@@ -1228,7 +1225,11 @@ public List<String> getUnValidatedEventFormSubmissionIds(int limit) {
             if (cursor != null && cursor.getCount() > 0 && cursor.moveToFirst()) {
                 while (!cursor.isAfterLast()) {
                     String id = cursor.getString(0);
-                    ids.add(id);
+
+                    if (StringUtils.isNotBlank(id))
+                        ids.add(id);
+                    else
+                        Timber.e("Unvalidated Event FormSubmissionId is empty/NULL: %s", cursor.getString(0));
```

Comment:
Let's remove `cursor.getString(0)` since we are only fetching one column in the SQL